server/ngwmn/views.py

```diff
@@ -53,17 +52,15 @@ def site_page(agency_cd, location_id):
 
     """
 
-    root = get_iddata('well_log', agency_cd, location_id)
```

Comment:
When I added the `get_well_log` function, I didn't think to remove the `get_iddata` call here.